tests/library/chromium/chromium.spec.ts

```diff
@@ -233,6 +233,68 @@ test('should intercept only serviceworker request, not page', async ({ context,
+  const contextRouted = [];
+  const pageRouted = [];
+  await context.route('**', async route => {
+    contextRouted.push(route.request());
+    await route.continue();
+  });
+  await page.route('**', async route => {
+    pageRouted.push(route.request());
+    await route.continue();
+  });
+  const contextRequests = [];
+  const pageRequests = [];
+  context.on('request', r => contextRequests.push(r));
+  page.on('request', r => pageRequests.push(r));
+
+  const [ sw ] = await Promise.all([
+    context.waitForEvent('serviceworker'),
+    page.goto(server.PREFIX + '/serviceworkers/doc-example/index.html'),
+  ]);
+
+  await expect.poll(() => sw.evaluate(() => (self as any).registration.active?.state)).toBe('activated');
+
+  await page.evaluate(() => fetch('./addressbook.json'));
+  await page.evaluate(() => fetch('./example.jpg'));
+  await page.evaluate(() => fetch('./tracker.js'));
+  await page.evaluate(() => fetch('./fallthrough.txt'));
+
+  expect(await Promise.all(contextRequests.map(formatRequest))).toEqual([
+    '[FRAME] index.html                fromServiceWorker: false',
+    '[SW   ] service-worker-main.js    fromServiceWorker: false',
+    '[SW   ] addressbook.json          fromServiceWorker: false',
+    '[FRAME] addressbook.json          fromServiceWorker: false',
+    '[FRAME] example.jpg               fromServiceWorker: false',
+    '[SW   ] example.jpg               fromServiceWorker: false',
+    '[FRAME] tracker.js                fromServiceWorker: false',
+    '[FRAME] fallthrough.txt           fromServiceWorker: false',
+    '[SW   ] fallthrough.txt           fromServiceWorker: false',
+  ]);
+  expect(await Promise.all(pageRequests.map(formatRequest))).toEqual([
+    '[FRAME] index.html                fromServiceWorker: false',
+    '[FRAME] addressbook.json          fromServiceWorker: false',
+    '[FRAME] example.jpg               fromServiceWorker: false',
+    '[FRAME] tracker.js                fromServiceWorker: false',
+    '[FRAME] fallthrough.txt           fromServiceWorker: false',
+  ]);
+  expect(await Promise.all(contextRouted.map(formatRequest))).toEqual([
+    '[SW   ] service-worker-main.js    fromServiceWorker: false',
+    '[SW   ] addressbook.json          fromServiceWorker: false',
+    '[SW   ] example.jpg               fromServiceWorker: false',
+    '[SW   ] fallthrough.txt           fromServiceWorker: false',
+  ]);
+  expect(await Promise.all(pageRouted.map(formatRequest))).toEqual([
+    '[FRAME] index.html                fromServiceWorker: false',
+    '[FRAME] addressbook.json          fromServiceWorker: false',
+    '[FRAME] example.jpg               fromServiceWorker: false',
+    '[FRAME] tracker.js                fromServiceWorker: false',
+    '[FRAME] fallthrough.txt           fromServiceWorker: false',
+  ]);
+});
```

Comment:
So `fromServiceWorker` seems to (1) be wrong here, (2) disagree with the docs, (3) disagree with out other tests.

I'm a bit confused though, since we have other cases of it working:

https://github.com/microsoft/playwright/blob/3436e64b755776108907c6e365dac86dffd1f6be/tests/library/chromium/chromium.spec.ts#L378

https://github.com/microsoft/playwright/blob/3436e64b755776108907c6e365dac86dffd1f6be/tests/library/chromium/chromium.spec.ts#L421

Raw CDP logs for this test look like it's always `false`.